flatdata-rs/lib/src/rawdata.rs

```diff
@@ -0,0 +1,35 @@
+/// Exposes blocks of raw data, providing auxiliary functionality like
+/// extracting substrings
```

Comment:
. is missing ;)

Reply:
Hmpf, if it were for me they would be removed everywhere :-)